pkgs/tools/networking/uwimap/default.nix

```diff
@@ -29,8 +29,8 @@ stdenv.mkDerivation {
     "-I${openssl.dev}/include/openssl";
 
   installPhase = ''
-    mkdir -p $out/bin $out/lib $out/include
-    cp c-client/*.h c-client/linkage.c $out/include
+    mkdir -p $out/bin $out/lib $out/include/c-client
+    cp c-client/*.h c-client/linkage.c $out/include/c-client/
```

Comment:
Is `linkage.c` really wanted in `$out/include`?

Reply:
Yes, amongst `auths.c` thats intended . I updated the PR.